lib/src/main/java/uk/gov/di/ipv/cri/address/library/service/AddressService.java

```diff
@@ -85,6 +85,16 @@ private ObjectReader getAddressReader() {
         return this.addressReader;
     }
 
+    public void setAddressCountryIfMissing(List<CanonicalAddress> addresses) {
+        for (var address : addresses) {
+            if (Objects.nonNull(address)) {
```

Comment:
Code smell recommends -> `Objects.nonNull(address) && Object.isNull(address.getAddressCountry()`

Reply:
This might be nicer

```
addresses.stream()
    .filter(Objects::nonNull)
    .filter(address -> Object.isNull(address.getAddressCountry()))
    .forEach(address -> address.setAddressCountry("GB"));
```

Reply:
Good call!